cmd/ctr/commands/snapshots/snapshots.go

```diff
@@ -207,7 +208,7 @@ var usageCommand = cli.Command{
 		var displaySize func(int64) string
 		if context.Bool("b") {
 			displaySize = func(s int64) string {
-				return fmt.Sprintf("%d", s)
+				return strconv.Itoa(int(s))
```

Comment:
nit: for `int64`, maybe use `strconv.FormatInt(s, 10)` directly, since [`Itoa` = `FormatInt(int64(s), 10)`](https://pkg.go.dev/strconv#Itoa)

Reply:
Thanks! Yes I was on the fence (thought perhaps this was slightly more readable), but I agree it's probably still readable enough; let me update

Reply:
Updated 👍